lib/workers/repository/process/lookup/types.ts

```diff
@@ -44,6 +44,8 @@ export interface LookupUpdateConfig
   datasource: string;
   depName: string;
   minimumConfidence?: string;
+  constraints?: Record<string, string>;
```

Comment:
This property is present in the `PackagFile` interface, its not in the `RenovateConfig` and its not extending it.

Reply:
`LookupUpdateConfig` is extending `RenovateConfig` and `constraints` should be defined there, as it's a user config property

Reply:
Not sure if I understand correctly but I'm accessing the `constraints` property here : 
https://github.com/renovatebot/renovate/blob/628327525ef4e1e7f30629db64b1b175b0077ca9/lib/workers/repository/process/lookup/common.ts#L7-L10
It will be an unknown type if I don't add it to the `LookupUpdateConfig` interface. Can you please explain a bit more ?

Reply:
yes, i remember that there are wome type issues when extending `RenovateConfig`, I'll try to prepare a type fix pr now